packages/material-ui/src/TextField/TextField.test.js

```diff
@@ -32,15 +32,19 @@ describe('<TextField />', () => {
     });
 
     it('should forward the multiline prop to Input', () => {
-      const wrapper = mount(<TextField multiline />);
+      const inputClasses = getClasses(<Input />);
+      const { getAllByRole } = render(<TextField multiline />);
 
-      expect(wrapper.find(Input).props()).to.have.property('multiline', true);
+      expect(getAllByRole('textbox')[0]).to.have.class(inputClasses.inputMultiline);
```

Comment:
Just curious would getting the non hidden text box work here instead?

Reply:
Yep, fixed!